lib/mdx.js

```diff
@@ -60,7 +60,7 @@ export async function getFileBySlug(type, slug) {
     source,
     // mdx imports can be automatically source from the components directory
     cwd: path.join(root, 'components'),
-    xdmOptions(options, frontmatter) {
+    mdxOptions(options, frontmatter) {
```

Comment:
Maybe it changed in newer versions, but the `xdmOptions` option is actually correct. 
<img width="734" alt="image" src="https://user-images.githubusercontent.com/1671563/202917591-6a3c880c-44a3-4880-acc9-3ffbaab23999.png">